src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,46 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX = 10;
+    private final Object[] keys;
+    private final Object[] values;
+    private int size;
+
+    public StorageImpl() {
+        keys = new Object[MAX];
+        values = new Object[MAX];
+    }
```

Comment:
We need to use class parametrization to avoid type mismatch
```suggestion
    private static final int MAX = 10;
    private final K[] keys;
    private final V[] values;
    private int size;

    public StorageImpl() {
        keys = (K[]) new Object[MAX];
        values = (V[]) new Object[MAX];
    }
```